src/library/simplified/extension/country-name.js

```diff
@@ -19,7 +19,12 @@ else if((utag.isCarSR() || utag.isPCarSearch()) && typeof b.entity.carSearch.sea
 }
 
 else if(packageSearchOrInfosite  && b.entity.packageSearch.packageSearchParameters.flightSearchParameters.arrivalAirportCityState != undefined) {
-	b["country"] = b["entity.packageSearch.packageSearchParameters.flightSearchParameters.arrivalAirportCityState"].split(",")[2]
+	var ctry = b.entity.packageSearch.packageSearchParameters.flightSearchParameters.arrivalAirportCityState.split(",");
+	if (ctry.length == 3) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/21168911/26581866/538f1708-455c-11e7-860a-cfdc9b977ede.png)

Repro URL
https://www.expedia.com/Hotel-Search?packageType=fh#destination=Las+Vegas+%28and+vicinity%29%2C+Nevada%2C+United+States+Of+America&startDate=6%2F23%2F2017&endDate=6%2F24%2F2017&adults=1&sort=recommended&searchProduct=hotel&c=77bfe9dd-56fd-4e22-b298-480ad8e6b8dd&ttla=LAS&packageType=fh